analytics-service/src/main/proto/patient_event.proto

```
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package patient.events;
```

Buf (1.55.1)

Files with package "patient.events" must be within a directory "patient/events" relative to root but were in directory "analytics-service/src/main/proto".

(PACKAGE_DIRECTORY_MATCH)